src/main/java/com/robertx22/mine_and_slash/aoe_data/database/exile_effects/adders/ModEffects.java

```diff
@@ -105,8 +107,7 @@ public static List<EffectCtx> getCurses() {
 
     // ---------- Helper ----------
     private static EffectCtx state(String id, String name, Elements elem) {
```

Comment:
This has to be done as method overload
```suggestion
    private static EffectCtx state(String id, String name, Elements elem) {
        return new EffectCtx(id, name, elem, EffectType.beneficial);
    }
    private static EffectCtx statePhysical(String id, String name) {
        return state(id, name, Elements.Physical);
    }
```